readme.md

```diff
@@ -403,10 +406,9 @@ import 'katex/dist/katex.min.css' // `rehype-katex` does not import the CSS for
 
 ReactDom.render(
   <ReactMarkdown
-    children={`The lift coefficient ($C_L$) is a dimensionless coefficient.`}
     remarkPlugins={[remarkMath]}
     rehypePlugins={[rehypeKatex]}
-  />,
+  >{`The lift coefficient ($C_L$) is a dimensionless coefficient.`}</ReactMarkdown>,
```

Comment:
```suggestion
  >
    {`The lift coefficient ($C_L$) is a dimensionless coefficient.`}
  </ReactMarkdown>,
```

Reply:
This can be plain text without template string?

```jsx
  <ReactMarkdown
    remarkPlugins={[remarkMath]}
    rehypePlugins={[rehypeKatex]}
  >
    The lift coefficient ($C_L$) is a dimensionless coefficient.
  </ReactMarkdown>
```

Reply:
It could be, but the spacing rules for JSX often lead to confusion. By this I don’t mean to say we shouldn’t use that, just that there is an argument to make for not doing it.